src/main/java/SasDefinitionSample.java

```diff
@@ -1,230 +1,215 @@
+import com.azure.security.keyvault.keys.models.KeyCurveName;
+import com.azure.security.keyvault.keys.models.KeyOperation;
+import com.azure.security.keyvault.keys.models.KeyType;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.sas.BlobSasPermission;
+import com.azure.storage.blob.sas.BlobServiceSasSignatureValues;
+import com.azure.storage.blob.specialized.BlockBlobClient;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import com.azure.storage.common.sas.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.UnsupportedEncodingException;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+public class SasDefinitionSample extends KeyVaultSampleBase {
+    private String STORAGE_ACCOUNT_NAME;
+    private String VAULT_URI;
+    private String VAULT_NAME;
+    private StorageAccount storageAccount;
+    private Vault vault;
+
+    public SasDefinitionSample() {
+        super();
+        setUpStorageAccount();
+    }
+
+
+    /**
+     * Creates an account sas definition, to manage storage account and its entities.
+     */
+    public void createAccountSasDefinition() throws UnsupportedEncodingException {
+        // In order to create an account sas definition, we have to first create a template. The
+        // template_uri for an account sas definition is the intended sas token signed with an arbitrary key.
+        // We create the generateSharedAccessSignature method in CloudStorageAccount to generate an account sas token.
+        StorageSharedKeyCredential sas = new StorageSharedKeyCredential(STORAGE_ACCOUNT_NAME, storageAccount.getKeys().get(0).value());
+
+        //Create a new policy
+        AccountSasService accountSasService = AccountSasService.parse("bfqt");
+        AccountSasResourceType accountSasResourceType = AccountSasResourceType.parse("sco");
+        AccountSasPermission accountSasPermission = AccountSasPermission.parse("rwdlacup");
+        AccountSasSignatureValues accountSasSignatureValues =
+                new AccountSasSignatureValues(
+                        OffsetDateTime.now().plusDays(1),
+                        accountSasPermission,
+                        accountSasService,
+                        accountSasResourceType);
+        //Generate a signature based off of the policy and account.
+        com.azure.storage.blob.BlobServiceClient blobServiceClient
+                = new com.azure.storage.blob.BlobServiceClientBuilder()
+                .endpoint(storageAccount.endPoints().primary().blob())
+                .credential(sas).buildClient();
+        String sasSignature = blobServiceClient.generateAccountSas(accountSasSignatureValues);
+        System.out.println("Generated sasSignature " + sasSignature);
+
+        // Generating new cloud storage account object off of the new acctSasToken
+        blobServiceClient = new com.azure.storage.blob.BlobServiceClientBuilder()
```

Comment:
Let's import this instead of using the fully qualified name.
```suggestion
        blobServiceClient = new BlobServiceClientBuilder()
```